Update contributing.md (#7)
equinor/videx-wellog: contributing.md

* **Videx Team Lead**: Maja Irina Wegmann (mweg@equinor.com)
* **Videx Tech Lead**: Kjerand Pedersen (kjerp@equinor.com)

## Contributing

In general, we follow the "fork-and-pull" Git workflow.

1. Fork the repo on GitHub
2. Clone the project to your own machine
3. Work on your fork
    1. Make your changes and additions

>         - Most of your changes should be focused on `src/` and `test/` folders and/or `README.md`, if applicable, please provide an example under 'examples/src'.

Reviewer: ```suggestion
        - Most of your changes should be focused on `src/` and `test/` folders and/or `README.md`, if applicable, please provide an example under `examples/src`.
```

* **Videx Team Lead**: Maja Irina Wegmann (mweg@equinor.com)
* **Videx Tech Lead**: Kjerand Pedersen (kjerp@equinor.com)

## Contributing

In general, we follow the "fork-and-pull" Git workflow.

1. Fork the repo on GitHub
2. Clone the project to your own machine
3. Work on your fork
    1. Make your changes and additions

>         - Most of your changes should be focused on `src/` and `test/` folders and/or `README.md`. 

Reviewer: should also include a line about adding an example if necessary under `examples/src`

>         - Files such as `validator.js`, `validator.min.js` and files in `lib/` folder are autogenerated when running tests (`npm test`) and need not to be changed **manually**. 

Reviewer: remove this

>         - Files such as `validator.js`, `validator.min.js` and files in `lib/` folder are autogenerated when running tests (`npm test`) and need not to be changed **manually**. 

Reviewer: ```suggestion
```

>         - Files such as `validator.js`, `validator.min.js` and files in `lib/` folder are autogenerated when running tests (`npm test`) and need not to be changed **manually**. 

Reviewer: ```suggestion
        - Most of your changes should be focused on `src/` and `test/` folders and/or `README.md`, if applicable, please provide an example under `examples/src`. 
```

>         - Files such as `validator.js`, `validator.min.js` and files in `lib/` folder are autogenerated when running tests (`npm test`) and need not to be changed **manually**. 

Reviewer: ```suggestion
        - Most of your changes should be focused on `src/` and `test/` folders and/or `README.md`, if applicable, please provide an example under `examples/src`. 
```

Reviewer: github is acting out, so I can't propose suggestions that can be merged
